spec/requests/api/connect/base_controller_spec.rb

```diff
@@ -55,15 +55,6 @@ def require_product
     end
   end
 
-  shared_examples 'updates the system token' do
```

Comment:
I'd suggest to move this to 'PUT /systems' for example, to make sure the token gets rotated correctly there.

Reply:
I have added test cases for read and write api's both.